### PR TITLE
Comment out default Consul check; Update URLs

### PR DESCRIPTION
**Pull request by angrycub**
Having an active check in the sample job causes issues with testing deployments in environments that are not integrated with Consul. This negatively impacts some of the getting-started experiences. Commenting out the check allows deployments to proceed successfully but leaves it in the sample job for convenience.

Made a drive-by fix to all of the URLs in the jobfile comments

### CODE REVIEW

**Comment by angrycub** [comment]
I did a really gross validation of the URLs with 

```
for I in $(git diff HEAD~2 | grep + | grep https | tr -d "+# " | sort | uniq); do curl -I -s $I; done
```

First pass revealed that there was one that was still being redirected via netlify.

File: command/assets/example.nomad [hunk 1]
```diff
@@ -327,19 +327,25 @@ job "example" {
...
-        check {
-          name     = "alive"
-          type     = "tcp"
-          interval = "10s"
-          timeout  = "2s"
-        }
+        # The "check" stanza instructs Nomad to create a Consul health check for
+        # this service. A sample check is provided here for your convenience;
+        # uncomment it to enable it. The "check" stanza is documented in the
+        # "service" stanza documentation.
```
Comment (tgross) [inline command/assets/example.nomad:340]: We're linking all over the place in this doc anyways... should we have a link to that doc page here too?

Comment (angrycub) [reply command/assets/example.nomad:340]: Do you mean linking to `service` or to `check`?  If to service, there's a link pretty close at the top of the service stanza itself, would that be adequate?  For linking to check, it's inside of the service page at a generated ID.  We _could_ link there, but I'm more worried about link rot on #ids.

Comment (tgross) [reply command/assets/example.nomad:340]: Oh, I missed that `service` link. Let's leave it as-is.

**Comment by tgross** [comment]
I've updated the compiled static assets to be current with `master` and this change in f188496

**Pull request merged**